docs/FAQ.md

```diff
@@ -10,6 +14,14 @@ We have user-contributed translations live in some languages. We intend to local
 
 If you are interested in contributing to translations please makes sure you [read this guide](how-to-translate-files.md) first.
 
+### Can I contribute articles to freeCodeCamp News or videos to freeCodeCamp's YouTube channel?
+
+Yes - you can contribute to our publication blog and YouTube channel.
+
+If you're interested in writing articles for freeCodeCamp News, please visit this[publication guide](https://www.freecodecamp.org/news/how-to-write-for-freecodecamp/). In addition, please read our[style guide](https://www.freecodecamp.org/news/developer-news-style-guide/) as this will help you write stronger and more effective articles.
```

Comment:
```suggestion
If you're interested in writing articles for freeCodeCamp News, please visit this [publication guide](https://www.freecodecamp.org/news/how-to-write-for-freecodecamp/). In addition, please read our [style guide](https://www.freecodecamp.org/news/developer-news-style-guide/) as this will help you write stronger and more effective articles.
```

missing spaces

Reply:
Thanks for catching that!